scripts/dashboard/WalletBalance.vue

```diff
@@ -141,6 +188,34 @@ getEventEmitter().on(
                     class="col-6 d-flex dcWallet-topLeftMenu"
                     style="justify-content: flex-start"
                 >
+                    <h3 class="noselect balance-title">
+                        <span
+                            class="reload"
+                            v-if="isViewOnly && isEncrypted && isImported"
+                            @click="restoreWallet()"
```

Comment:
You need to define an event, emit it and restore in Dashboard.vue because the function doesn't exist in walletbalance

Reply:
Oh that makes totally sense!